src/main/accumulMod.F90

```diff
@@ -717,6 +735,7 @@ subroutine accumulRest( ncid, flag )
                dim1name=accum(nf)%type1d, dim2name=accum(nf)%type2d, &
                long_name='number of accumulated steps for '//trim(accum(nf)%name), &
                units='-', &
+               switchdim=.false., scale_by_thickness=accum(nf)%scale_by_thickness, &
```

Comment:
Hmmm, I'm not sure what's the right thing to do here: should NSTEPS actually be scaled by thickness? But then again, I'm not sure it makes sense to do a vertical interpolation on NSTEPS in the first place. We're probably saved by the fact that NSTEPS will probably always be the same for all levels, so the vertical interpolation will be trivial. But maybe it's worth adding a comment like this to call our attention to this oddity for the future?

More generally: thank you for your changes to accumulMod. They look good to me (other than the minor issue I noted above).